CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 1.1.10 - 2017-7-7
+This is a feature release.
+
+Feature:
+  * (RE-8861) Generate metadata files when building projects
```

Comment:
We should put a period at the end of this sentence.

Since this is rendered as Markdown it doesn't put a newline here, so this turns into a run-on sentence like:
`(RE-8861) Generate metadata files when buildings projects Adds a new 'manifest' command....`